### ISSUE TITLE
User Impersonation

### ISSUE THREAD

**Issue by bencap** [VariantEffect/mavedb-api#182]
In some cases, it is useful for certain classes of user to be able to impersonate another user. We should add this role/ability.

**Comment by afrubin** [comment]
What is an example use case for this functionality? If we have an admin role that allows the admin to edit any record as if they were the owner, it seems like that would cover many of the uses here.

Are you thinking that it would be helpful to be able to see a user's dashboard view (for example) when providing support?

**Comment by jstone-uw** [comment]
Yes, that's the type of use case we had in mind in discussing this. Most things should be feasible as an administrator, but for support purposes it's often helpful to see what a user sees.
